tests/rules/javascript/test.no_eval.js

```diff
@@ -24,13 +24,11 @@ describe('no_eval', () => {
   ];
 
```

Comment:
I've checked source codes with the ones in the file. The additional codes that are present in the source code turned out to be invalid codes in test.no_implied_eval.js so I haven't added them.